lib/active_hash/base.rb

```diff
@@ -50,6 +50,10 @@ def field_names
         @field_names ||= []
       end
 
+      def column_names
+        @column_names ||= field_names.map(&:to_s)
```

Comment:
```suggestion
        @column_names ||= field_names.map(&:name)
```


Please use `Symbol.name` to avoid creating a new string for every invocation.

```
irb(main):005> :foo.to_s.frozen?
=> false
irb(main):006> :foo.to_s.object_id
=> 63900
irb(main):007> :foo.to_s.object_id
=> 65360
irb(main):008> :foo.to_s.object_id
=> 66820
```

vs

```
irb(main):016> :foo.name.frozen?
=> true
irb(main):017> :foo.name.object_id
=> 75280
irb(main):018> :foo.name.object_id
=> 75280
irb(main):019> :foo.name.object_id
=> 75280
```

Reply:
Got it! I didn't know that. I fix it ⚒️
https://github.com/active-hash/active_hash/pull/311/commits/0540d0301c9129ceae7fa56af1e8d2696ee44f4b

Reply:
Ruby versions prior to 3.0 do not have the name method in the Symbol class. CI Fail [Job].(https://github.com/active-hash/active_hash/actions/runs/9796570215/job/27117706794) 

So, I fix it again https://github.com/active-hash/active_hash/pull/311/commits/2f6b35d4a0c5f21ec5365bb36668cb12be5b7551